views/update/update-interrupt-card.html

```diff
@@ -31,6 +31,7 @@ <h2 class="govuk-heading-m">Check if any information needs to be updated</h2>
             If you need to update any information, you can do that as part of this service. Any changes must have been <a href="https://www.gov.uk/guidance/remove-an-overseas-entity#verify" class="govuk-link" data-event-id="verified-agent-link">verified by a UK-regulated agent</a> no more than 3 months before this application is submitted.
           </p>
           <form method="post">
+            {% include "includes/csrf_token.html" %}
```

Comment:
does this need a test updating? i couldn't see it on this PR

Reply:
Its a shared Update page. The test was updated as part of the update PR